src/components/left-section/LeftSection.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { NavLink } from "react-router-dom";
+
+export function LeftSection({ addPostShowModal, showModal, setShowModal }) {
+  return (
+    <div className="left-section">
+      <NavLink className="link-no-style sidebar-item" to="/home">
+        <span class="material-symbols-outlined">home</span>
+        <span>Home</span>
+      </NavLink>
```

Comment:
suggestion: you put the data in an array and just loop over it

Reply:
Done